modules/core/src/shaderlib/shadow/shadow.ts

```diff
@@ -158,7 +158,7 @@ function getViewProjectionMatrices({
     [0, viewport.height, -1], // bottom left near
     [viewport.width, viewport.height, -1] // bottom right near
   ].map(pixel =>
-    // ts-expect-error z may be undefined
+    // @ts-expect-error z may be undefined
```

Comment:
There is no `z` variable used here. Maybe clarify, perhaps it is length of array or...

Reply:
Just reverting to what already exists in v8